src/main/java/com/my/FoodTruckBackend/customer/CustomerService.java

```diff
@@ -0,0 +1,34 @@
+package com.my.FoodTruckBackend.customer;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class CustomerService {
+    private final CustomerRepository customerRepository;
+
+    public List<CustomerDto> getListOfCustomers() {
+        return customerRepository.getListOfCustomers();
+    }
+
+    public CustomerDto createNewCustomers(NewCustomerRequestBody newCustomerRequestBody) {
+        if (newCustomerRequestBody.getPassword().equals(newCustomerRequestBody.getConfirmPassword())) {} else {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "passwords do not match");
+        }
```

Comment:
we should be able to do the opposite of what this says, to get rid of an `else` statement

Reply:
You're correct, this has been simplified